l10n_it_fatturapa_sale/models/account_invoice.py

```diff
@@ -9,7 +9,11 @@ class AccountInvoice (models.Model):
 
     @api.multi
     def unlink(self):
-        related_documents = self.mapped('related_documents')
+        # Use sudo because current user might not be able to
+        # read the related documents
+        # but they should be unlinked just the same
+        self_sudo = self.sudo()
+        related_documents = self_sudo.mapped('related_documents')
         res = super().unlink()
         related_documents.check_unlink().unlink()
```

Comment:
E un più diretto?
```suggestion
        self.sudo().mapped('related_documents').check_unlink().unlink()
```

Reply:
Grazie della review!
Va bene lo stesso se lascio com'è?
Credo che mettere tutto in una riga riduca la leggibilità, e non mi sembra che il codice attuale possa creare problemi.

Reply:
Ok